test/spec/extension.js

```diff
@@ -124,6 +124,46 @@ describe('extension - can copy', function() {
       expect(canCopyProperty).to.be.false;
     });
 
+
+    it('should allow on intermediateThrowEvent with messageDefinition', function() {
+
+      // given
+      var taskDefinition = moddle.create('zeebe:LoopCharacteristics'),
```

Comment:
This is not a taskDefinition?
```suggestion
      var loopCharacteristics = moddle.create('zeebe:LoopCharacteristics'),
```